src/components/Cards.tsx

```diff
@@ -2,14 +2,14 @@ import React, { useContext } from "react";
 
 import { ACTION } from "../context/Actions";
 import { GlobalContext } from "../context/GlobalState";
-import { Element, NumberType, CartItem } from "../context/types";
+import { NumberType, CartItem } from "../context/types";
 
 function Cards(): JSX.Element {
   const { state, dispatch } = useContext(GlobalContext);
 
   return (
     <React.Fragment>
-      {state.data.map((element: Element, index: NumberType) => (
+      {state.res.map((element: any, index: NumberType) => (
```

Comment:
Use of `any` is discouraged here. Please use a specific data type for the object.